ci/jenkins/jobs/macros.yaml

```diff
@@ -293,3 +293,52 @@
           set -e
           DOCKER_REGISTRY="$(head -n1 ci/docker-registry)"
           ./ci/jenkins/test.sh --testcase '{conformance_type}' --registry ${{DOCKER_REGISTRY}} --testbed-type jumper
+
+- builder:
+    name: builder-capa-matrix
+    builders:
+      - shell: |-
+          #!/bin/bash
+          set -ex
+          DOCKER_REGISTRY="$(head -n1 ci/docker-registry)"
+          export JOB_NAME="matrix-${TEST_OS}-k8s-${K8S_VERSION//./-}-build-num"
+          ./ci/jenkins/test-vmc.sh --cluster-name "${JOB_NAME}-${BUILD_NUMBER}" --setup-only --provider aws --aws-region "us-west-2" --aws-access-key-id "${AWS_ACCESS_KEY}" --aws-secret-access-key "${AWS_SECRET_KEY}" --aws-service-user-name "${AWS_SERVICE_USER_NAME}" --aws-service-user-role "${AWS_SERVICE_USER_ROLE_ARN}" --aws-vpc-id "${CAPA_VPC}" --aws-subnet-id "${CAPA_SUBNET}" 
+          testcases=("e2e" "conformance" "networkpolicy")
+          for testcase in "${testcases[@]}"; do
+            ./ci/jenkins/test-vmc.sh --cluster-name "${JOB_NAME}-${BUILD_NUMBER}" --testcase "$testcase" --test-only --provider aws --aws-region "us-west-2" --aws-access-key-id "${AWS_ACCESS_KEY}" --aws-secret-access-key "${AWS_SECRET_KEY}" --aws-service-user-name "${AWS_SERVICE_USER_NAME}" --aws-service-user-role "${AWS_SERVICE_USER_ROLE_ARN}" --aws-vpc-id "${CAPA_VPC}" --aws-subnet-id "${CAPA_SUBNET}"
+            test_result=$?
+            echo "$test_result" > "test-$testcase-rc.txt"
+          done
+          ./ci/jenkins/test-vmc.sh --cluster-name "${JOB_NAME}-${BUILD_NUMBER}" --cleanup-only --provider aws
+          failure=0
+          for result in test-*-rc.txt; do
+            r="$(cat $result)"
+            if [ "$r" -ne 0 ]; then
+              echo Test Failed: $result : $r
+              failure=failure+1
+            fi
+          done
+          exit failure
+
+- builder:
+    name: builder-capa-aws-clean-up
+    builders:
+      - shell: |-
+          #!/bin/bash
+          for clustername in `kubectl get clusters -A -l jenkins-test --no-headers | grep test | awk '{print $1}'`; do 
+              echo $clustername;
+              timestamp="$(kubectl get clusters -o jsonpath="{.metadata.creationTimestamp}" $clustername -n $clustername)"
```

Comment:
prefer to use name like cluster_creation_timestamp, instead of generic names like timestamp